src/network/network_chat_gui.cpp

```diff
@@ -58,17 +65,20 @@ static PointDimension _chatmsg_box;
 static uint8 *_chatmessage_backup = nullptr; ///< Backup in case text is moved.
 
 /**
- * Count the chat messages.
- * @return The number of chat messages.
+ * Test if there are any chat messages to display.
+ * @param show_all Set if all messages should be included, instead of unexpired only.
+ * @return True iff there are chat messages to display.
```

Comment:
```suggestion
 * @return True if there are chat messages to display.
```
:)

Reply:
"Usage of the abbreviation "iff" first appeared in print in John L. Kelley's 1955 book General Topology.[14] Its invention is often credited to Paul Halmos, who wrote "I invented 'iff,' for 'if and only if'—but I could never believe I was really its first inventor.""

:)